package.json

```diff
@@ -52,19 +52,32 @@
     "publish:rc": "npm publish dist/clr-ui --tag rc; npm publish dist/clr-angular --tag rc; npm publish dist/clr-icons --tag rc",
     "publish:next": "npm publish dist/clr-ui --tag next; npm publish dist/clr-angular --tag next; npm publish dist/clr-icons --tag next",
     "publish:local": "npm publish dist/clr-ui --registry http://localhost:4873; npm publish dist/clr-angular --registry http://localhost:4873; npm publish dist/clr-icons --registry http://localhost:4873",
-    "ks:publish": "ng build --prod ks-app; cat ./dist/ks-app/index.html > ./dist/ks-app/404.html; surge -p ./dist/ks-app"
+    "ks:publish": "ng build --prod ks-app; cat ./dist/ks-app/index.html > ./dist/ks-app/404.html; surge -p ./dist/ks-app",
+    "compile:server": "webpack --config src/website/webpack.server.config.js --progress --colors",
+    "serve:ssr": "node dist/website/server",
+    "build:ssr": "ng build website --prod && ng run website:server:production && npm run compile:server",
```

Comment:
we should rename these before the final merge. We can tell they are website related by looking at it, but I was initially looking for the `website:` prefixes on these (`website:compile:server` etc).